components/motor/gpio/motor_encoder.go

```diff
@@ -378,6 +379,10 @@ func (m *EncodedMotor) DirectionMoving() int64 {
 func (m *EncodedMotor) directionMovingInLock() float64 {
 	move, err := m.real.IsMoving(context.Background())
 	if move {
+		if sign(m.state.lastPowerPct) != sign(m.state.direction) {
+			// short sleep when changing directions to minimize lost ticks in single encoder
+			time.Sleep(10 * time.Microsecond)
```

Comment:
this is unideal but it greatly improved the behavior with single encoders

Reply:
lmk if you have better ideas on how to accomplish this. tried using a `SelectContextOrWait` but that did not improve the behavior

Reply:
hmm don't have any ideas beside sleep but does it need 10 microseconds to improve or would something smaller also work?

Reply:
I tried 5 and it wasn't better unfortunately